webapp/store/snap_details_views.py

```diff
@@ -352,6 +352,7 @@ def snap_details_badge(snap_name):
 
     @store.route('/<regex("' + snap_regex + '"):snap_name>/trending.svg')
     def snap_details_badge_trending(snap_name):
+        is_preview = flask.request.args.get("preview", default=0, type=int)
```

Comment:
I don't like having a "preview" arg that anyone can use - we need to verify server-side imo.

Untested:

```suggestion
        can_preview = flast.request.headers.get("Host") == "snapcraft.io"
        is_preview = flask.request.args.get("preview", default=0, type=int) if can_preview
```